lib/system/arc.nim

```diff
@@ -60,7 +60,7 @@ elif defined(nimArcIds):
 when defined(gcAtomicArc) and hasThreadSupport:
   template decrement(cell: Cell): untyped =
     discard atomicDec(cell.rc, rcIncrement)
-  template increment(cell: Cell): untyped =
+  proc increment(cell: Cell) =
```

Comment:
Why?

Reply:
Sorry, I was debugging an issue and submitted the wrong code